flytepropeller/pkg/controller/nodes/catalog/datacatalog/transformer.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
+	"github.com/golang/protobuf/proto"
```

Comment:
does the linter not complain about this? i thought our convention would place this below the "github.com/flyteorg/..." imports.

Reply:
i thought so too